tests/microos/toolbox.pm

```diff
@@ -0,0 +1,96 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Run simple toolbox tests
+# Maintainer: Jose Lausuch <jalausuch@suse.com>
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
+use version_utils 'is_sle_micro';
+
+our $user = 'test_user';
+
+sub cleanup {
```

Comment:
isn't it duplication of https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/containers/common.pm#L128 ?

Reply:
Maybe, I can call `clean_container_host` from here but I need to keep `userdel -rf $user` anyways.

Reply:
3 options possible and I don't have strong opinion which one is better here ... : 
1. leave things as it is right now . 
2. call cleanup from `containers/common.pm` here to avoid duplication with podman part of cleanup
3. increase amount of `if`'s in  `containers/common.pm`  and merge altogether 

each choice has down and up sides so I am not sure ....

Reply:
Option 2 for now :)